test/controllers/publishers/promo_registrations_controller_test.rb

```diff
@@ -2,6 +2,10 @@
 require "webmock/minitest"
 require "shared/mailer_test_helper"
 
+# (Albert Wang): Because we disabled referrals, users can't create referrals for themselves
+# The controller's create function only runs in tests and in development.
+# Although it's unlikely, we retain this in case we need to turn back on the referral program.
```

Comment:
Just delete it. It will clean up the codebase and we can always restore from git history if we need to. We can delete the associated test as well, and this comment :)